tests/integration/defs/perf/open_search_db_utils.py

```
@@ -448,105 +456,90 @@ def parse_timestamp(timestamp):
     if threshold_key in baseline_data:
         return baseline_data[threshold_key]
 
-    # Fall back to general threshold
-    fallback_key = f"d_threshold_{metric_suffix}"
-    if fallback_key in baseline_data:
-        return baseline_data[fallback_key]
-
-    # No threshold found, raise error
     raise KeyError(
         f"No threshold found for metric '{metric}'. "
         f"Expected '{threshold_key}' or '{fallback_key}' in baseline data.")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)